hadoop-hdds/server-scm/src/main/java/org/apache/hadoop/hdds/scm/block/SCMBlockDeletingService.java

```diff
@@ -229,7 +229,7 @@ public String getServiceName() {
 
   @Override
   public void stop() {
-    throw new RuntimeException("Not supported operation.");
+    // Not supported operation
```

Comment:
Please keep the origin implementation.

Reply:
When call stop of serviceManager, as its registered part of serviceManager causes stop to fail. We must support stop as common framework or do nothing in stop.
So this is removed.

Reply:
In other words it's not an unsupported operation, rather a no-op.

`BlockManagerImpl` explicitly starts and shuts down `SCMBlockDeletingService`, which is also registered with `SCMServiceManager`, which manages start/stop of registered services.  I think it's confusing to have both of these lifecycle management calls for the same service.  Can we merge `stop` and `shutdown` and let only `SCMServiceManager` manage it?

Reply:
1. For this PR, will avoid serviceManager.stop() and scope out this cleanup
2. OR remove serviceManager.stop() and add explicit member fields of class and do cleanup as SCM stop
3. OR SCMBlockDeletingService stop() call internally shutdown()
Which process should we do currently?

Reply:
I checked, BlockManagerImpl is also calling close of blockingService twice (may be code bug but not in scope). But this has no impact.
Considering this, will call shutdown() in stop() of SCMBlockDeletingService, and this do not have any impact for code reuse.